### PR TITLE
Added option to specify a root path for IBFT validators dir

### PR DESCRIPTION
**Pull request by begmaroman**
# Description

Currently, the command that generates validators set does not support an option to use a specific root dir so the current one is used always. 
This PR introduces a root validators path parameter.

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [x] I have tested this code manually

### CODE REVIEW

File: docker/local/polygon-edge.sh [hunk 1]
```diff
@@ -46,7 +46,8 @@ case "$1" in
                   rm -f /data/genesis.json
 
                   createGenesisConfig "$2" "$secrets" \
-                  --ibft-validators-prefix-path data-
+                    --ibft-validators-path /data \
+                    --ibft-validators-prefix-path data-
```
Comment (Stefan-Ethernal) [inline docker/local/polygon-edge.sh]: WDYT about removing `ibft-validators-path` and `ibft-validators-prefix-path` flags from the `genesis` command and using `validators-path` and `validators-prefix` flags for both IBFT and Polybft? We can do that through a separate PR, if you find it possible.

File: command/genesis/genesis.go [hunk 2]
```diff
@@ -125,6 +125,14 @@ func setFlags(cmd *cobra.Command) {
 			"the type of validators in IBFT",
 		)
 
+		cmd.Flags().StringVar(
+			&params.validatorRootPath,
```
Comment (Stefan-Ethernal) [inline command/genesis/genesis.go]: Should we validate if the path exists and stop the execution otherwise?

File: command/ibft/switch/ibft_switch.go [hunk 3]
```diff
@@ -65,6 +65,14 @@ func setFlags(cmd *cobra.Command) {
 
 	{
 		// PoA Configuration
+		cmd.Flags().StringVar(
+			&params.ibftValidatorRootPath,
```
Comment (Stefan-Ethernal) [inline command/ibft/switch/ibft_switch.go]: Validate existence of a path?

**Review by Stefan-Ethernal — APPROVED** [review]
Overal LGTM, tnx for fixing this. Leaving a couple of considerations though.

File: docker/local/polygon-edge.sh (hunk 1 above)
Comment (begmaroman) [reply docker/local/polygon-edge.sh]: I'll do that within this PR

Comment (begmaroman) [reply docker/local/polygon-edge.sh]: Done

**Review by rachit77 — APPROVED** [review]

File: command/ibft/switch/ibft_switch.go (hunk 3 above)
Comment (begmaroman) [reply command/ibft/switch/ibft_switch.go]: Done

File: command/genesis/genesis.go (hunk 2 above)
Comment (begmaroman) [reply command/genesis/genesis.go]: It is done already: https://github.com/0xPolygon/polygon-edge/blob/develop/command/genesis/params.go#L206-L210

**Review by stana-miric — APPROVED** [review]
